parlai/tasks/integration_tests/agents.py

```diff
@@ -575,7 +577,7 @@ def load_from_chunk(self, chunk_idx: int):
             output.append((text, resp))
         return output
 
-    def create_message(self, sample_item):
+    def create_message(self, sample_item, entry_idx=0):
```

Comment:
it'd probably be nice to have a `MultiturnChunkyTeacher` as well in the integration tests 😄